### DISCUSSION TITLE
Does Turbopack work with next-pwa?

### DISCUSSION THREAD

**Discussion by moroshko** [vercel/turborepo#4314] [Help] [answered]
I tried to run the [Progressive Web App example](https://github.com/vercel/next.js/tree/canary/examples/progressive-web-app) with `next dev --turbo`, but getting an error:
```
Turbopack failed to compile
Error resolving commonjs request
```
<img width="973" alt="Screenshot 2023-03-23 at 4 05 40 pm" src="https://user-images.githubusercontent.com/259753/227109075-ec88ce5a-5840-4715-b90a-ee5b3c21ae30.png">

Does Turbopack work with `next-pwa`?

**Comment by anthonyshew** [comment, marked as answer]
If you're still having this problem, please open up an issue. 🙏
